code/modules/mob/living/carbon/human/human.dm

```diff
@@ -923,7 +923,7 @@
 	return ..()
 
 /mob/living/carbon/human/is_literate()
-	return TRUE
+	return !has_trauma_type(/datum/brain_trauma/mild/illiterate, TRAUMA_RESILIENCE_ABSOLUTE)
```

Comment:
I'd make this into a trait, so other sources can do the same as well

Reply:
good idea, I'll do this after work.

Reply:
Just so it's clear, making this a trait should also necessitate removing `is_literate` completely. I imagine this is used for things like making random simple animals illiterate or whatever, in which case you'd want to give them that trait.

Reply:
So, I have looked into changing literacy into a trait.

It would either involve sweeping changes to making most things literate, OR require a LOT of code to maintain current functionality.